<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayChart</class>
 <widget class="QMainWindow" name="DayChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Daily Chart</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sym_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Symbol</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_lbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sym_in">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_in">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="get_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Data</string>
    </property>
   </widget>
   <widget class="MplGraph" name="MplGraph" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>981</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuasdfsd">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave_data"/>
   </widget>
   <addaction name="menuasdfsd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_data">
   <property name="text">
    <string>Save data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplGraph</class>
   <extends>QWidget</extends>
   <header>mplgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
